config/secrets.yml

```diff
@@ -14,7 +14,7 @@ development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 7053b205c9559f83960f18d23f10d36cacd737c0e17e0dd71cbb90d2f3e6
```

Comment:
This key is not in the env for the build hook, and is required to run tests.
Doesn't seem like the worst thing in the world to have this "secret" be an arbitrary value and have in the the source repo. @bparees wdyt?